settings.gradle

```diff
@@ -229,6 +229,7 @@ includeProject(":lint-checks:integration-tests")
 
 includeProject(":core:core-bundle")
 includeProject(":core:core-uri")
+includeProject(":core:core-viewtree")
```

Comment:
Let's avoid adding projects that we do not support here.
1. We do not compile Android
2. If you really need this, please just use the published binary

Reply:
I wanted to keep the build scipt as close as possible to the AOSP version. I'll use published version then

Reply:
build gradles are a separate story. We need to sync buildSrc first to properly align it.